doc/requirements/sequential_cutoff

```diff
@@ -0,0 +1,164 @@
+from 1 to 4194181 kiB inclusive.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Default sequential cutoff threshold value
+id: threshold_default
+---
+
+The default sequential cutoff threshold shall be 1 MiB (1024 kiB).
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting sequential cutoff promotion count for core
+id: promotion_count_core
+---
+
+The management tool shall allow to specify sequential cutoff promotion count
+for given core.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: >-
+  Setting sequential cutoff promotion count for all cores in given
+  cache instance
+id: promotion_count_all_cores
+---
+
+The management tool shall allow to specify sequential cutoff promotion count
+for all cores associated with given cache instance at once.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Allowed sequential cutoff promotion count values
+id: promotion_count_values
+---
+
+It shall be possible to specify sequential cutoff promotion count value
+in range from 0 to 65535 inclusive.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Default sequential cutoff promotion threshold value
+id: promotion_count_default
+---
+
+The default sequential cutoff promotion threshold shall be 8.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Loading sequential cutoff configuration on cache load
+id: loading_configuration
+---
+
+All sequential cutoff parameters shall be loaded from cache metadata for each
+core on cache load.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Sequential cutoff operation
+id: operation
+---
+
+The sequential cutoff shall detect stream of sequential io requests and when
+amount of data within the stream reaches the cutoff threshold, each next io
+request which continues the stream shall be handled in Pass-Through mode.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Detecting multiple sequential cutoff streams
+id: detecting_multiple_streams
+---
+
+The sequential cutoff shall detect at least 64 separate streams of sequential
```

Comment:
at least 64? I thought that e.g. 2 streams will be detected too and the range of possible to detect streams is 1-256

Reply:
Fixed.